src/components/custom-field-input-text/custom-field-input-text.test.jsx

```diff
@@ -74,14 +74,15 @@ describe('CustomFieldInputText', () => {
   });
 
   describe('validations', () => {
-    it('initially, it does not show errors for html-invalidity. It shows errors for html-invalidity after user interaction', () => {
+    it('initially, it does not show errors for html-invalidity. It shows errors for html-invalidity after tabbing', () => {
       render(<CustomFieldInputText {...requiredProps} required />);
       expect(screen.getByLabelText('Test label')).toBeInvalid();
       expect(screen.getByLabelText('Test label')).toHaveDescription('');
       userEvent.type(screen.getByLabelText('Test label'), '123');
       expect(screen.getByLabelText('Test label')).toBeValid();
       expect(screen.getByLabelText('Test label')).toHaveDescription('');
       userEvent.type(screen.getByLabelText('Test label'), '{backspace}{backspace}{backspace}');
```

Comment:
Let's add an assertion that there is no description until the tab action. 
```suggestion
      userEvent.type(screen.getByLabelText('Test label'), '{backspace}{backspace}{backspace}');
      expect(screen.getByLabelText('Test label')).toHaveDescription('');
```